js/map.js

```diff
@@ -0,0 +1,83 @@
+import { generateAd } from './generate-ad.js';
+import { adForm, adSet, COORDINATES_TOKYO } from './data.js';
+import { togglePageActiveState } from './page-state.js';
+
+const PinSetting = {
+  MAIN: {
+    iconUrl: 'img/main-pin.svg',
+    iconSize: [52, 52],
+    iconAnchor: [26, 52],
+  },
+  REGULAR: {
+    iconUrl: 'img/pin.svg',
+    iconSize: [40, 40],
+    iconAnchor: [20, 40],
+  },
+};
+
+const mainPinIcon = L.icon(PinSetting.MAIN);
+
+const pinIcon = L.icon(PinSetting.REGULAR);
+
+const map = L.map('map-canvas')
+  .on('load', () => {
+    togglePageActiveState(false);
+  })
+  .setView({
+    lat: COORDINATES_TOKYO.lat,
+    lng: COORDINATES_TOKYO.lng,
+  }, 14);
```

Comment:
Ключи объекта совпадают, значит можно передавать объект напрямую:
```suggestion
  .setView(COORDINATES_TOKYO, 14);
```
Аналогично ниже